components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1727,8 +1727,11 @@ interface BraveWalletP3A {
 
 enum OnboardingAction {
   Shown = 0,
-  CreatedWallet = 1,
-  RestoredWallet = 2
+  LegalAndPassword = 1,
+  RecoverySetup = 2,
+  Complete = 3,
+  CompleteRecoverySkipped = 4,
+  StartRestore = 5
```

Comment:
Is it ok that both CreateWallet and LegalAndPassword are both reported as 1? Should we start LegalAndPassword as 3 and mark CreatedWallet and RestoredWallet as deprecated or so?

Reply:
nvm it is now Brave.Wallet.OnboardingConversion.3